credsweeper/filters/value_jwt_check.py

```diff
@@ -0,0 +1,49 @@
+import contextlib
+import json
+import math
+
+from credsweeper.common.constants import Chars
+from credsweeper.config import Config
+from credsweeper.credentials import LineData
+from credsweeper.file_handler.analysis_target import AnalysisTarget
+from credsweeper.filters import Filter, ValueEntropyBase64Check
+from credsweeper.utils import Util
+
+
+class ValueJwtCheck(Filter):
+    """JWT token check - simple"""
+
+    def __init__(self, config: Config = None) -> None:
+        pass
+
+    def run(self, line_data: LineData, target: AnalysisTarget) -> bool:
```

Comment:
The checker is very loose. It must be more strict, at least check the inner structure for the presence of some security-related keywords.